main.go

```diff
@@ -207,10 +208,17 @@ func main() {
 	mux.Handle("/mdm/connect", connectHandler)
 
 	if checkEmptyArgs(*flUrl, *flScepUrl) {
-		logger.Log("warn", "Enrollment endpoint /mdm/enroll will be disabled because you did not specify flags for the external URL or SCEP URL")
+		logger.Log("warn", "Enrollment endpoint /mdm/enroll will be disabled because you did not specify flags/environment vars for the external URL (--url MICROMDM_URL) or SCEP URL (--scep-url/MICROMDM_SCEP_URL)")
```

Comment:
Is one of these options required? If so ask the user to enter the appropriate flag and exit.